build

```diff
@@ -42,12 +42,20 @@ if [ -w ${go_root_dir}/pkg ]; then
 fi
 
 for cmd in sentinel proxy; do
+  echo "Building stolon-${cmd} ..."
 	CGO_ENABLED=0 go build -installsuffix cgo -ldflags "$LD_FLAGS" -o ${BINDIR}/stolon-${cmd} ${REPO_PATH}/cmd/${cmd}
+  echo "Done building stolon-${cmd}"
 done
+
+echo "Building stolonctl ..."
 CGO_ENABLED=0 go build -installsuffix cgo -ldflags "$LD_FLAGS" -o ${BINDIR}/stolonctl ${REPO_PATH}/cmd/stolonctl
+echo "Done building stolonctl"
 
-go build -ldflags "$LD_FLAGS" -o ${BINDIR}/stolon-keeper ${REPO_PATH}/cmd/keeper 
+echo "Building stolon-keeper ..."
+go build -ldflags "$LD_FLAGS" -o ${BINDIR}/stolon-keeper ${REPO_PATH}/cmd/keeper
+echo "Done building stolon-keeper"
 
+echo "Doing some final setup ..."
```

Comment:
Probably better explaining we are copying the binaries for docker images.

Reply:
Changed the message to say we are copying binaries for docker images